docs/buildah-rmi.md

```diff
@@ -0,0 +1,29 @@
+## buildah-rmi "March 2017"
+
+## NAME
+buildah rmi - Removes an image or images. 
```

Comment:
+buildah rmi - Removes one or more images.. 

Need a comment about what happens if the images are being used.